app/containers/Questions/saga.js

```diff
@@ -144,11 +146,28 @@ export function* getQuestionsWorker({
         followedCommunities,
       );
     }
-
+    const usersSet = new Set();
     questionsList.forEach(question => {
       question.isGeneral = isGeneralQuestion(question);
+      usersSet.add(question.author.id);
     });
 
+    //TODO delete after graph ready
+    yield all(
+      [...usersSet].map(function*(user) {
+        yield call(getUserProfileWorker, { user, getFullProfile: true });
+      }),
+    );
+    yield all(
+      questionsList.map(function*(question) {
+        const profileObject = yield select(selectUsers(question.author.id));
+        question.author = {
+          ...question.author,
+          ratings: profileObject.ratings,
+          highestRating: profileObject.highestRating,
+        };
+      }),
+    );
     // yield all(
```

Comment:
should we remove this commented code?

Reply:
This code is for future releases

Reply:
add TODO in this case, because just commented code without any highlights smells bad